src/components/matrix/Matrix.java

```diff
@@ -0,0 +1,117 @@
+package src.components.matrix;
+
+//Hello Jeremy
+
+public class Matrix {// extends components.standard<Matrix> {
+    private int[][] rep;
+
+    private void createNewRep(int i, int j) {
+        this.rep = new int[i][j];
+
+        for (int x = 0; x < i; x++) {
+            for (int y = 0; y < j; y++) {
+                this.rep[x][y] = 0;
+            }
+        }
+    }
+
+    public Matrix(int[][] m) {
+        this.rep = m.clone();
+    }
+
+    public Matrix(int i, int j) {
+        this.createNewRep(i, j);
+    }
+
+    public int get(int i, int j) {
+        return this.rep[i][j];
+    }
+
+    public void set(int i, int j, int n) {
+        this.rep[i][j] = n;
+    }
+
+    public void clear() {
+        this.createNewRep(this.rep.length, this.rep[0].length);
+    }
+
+    public int[] size() {
+        return new int[] { this.rep.length, this.rep[0].length };
+    }
+
+    public int[][] rep() {
```

Comment:
I would not expose this to the user for aliasing reasons. If you want to give the user a matrix, I would create a copy of the existing matrix.